src/iterative_ensemble_smoother/_iterative_ensemble_smoother.py

```diff
@@ -142,7 +143,7 @@ def fit(
 
         self.ensemble_mask = ensemble_mask
 
-        W: npt.NDArray[np.double] = create_coefficient_matrix(
+        W: npt.NDArray[np.double] = create_coefficient_matrix(  # type: ignore
```

Comment:
Why are we ignoring? Because new `ies` Python module is not annotated?

Reply:
Yes